lib/rules/sotd/mailing-list.js

```diff
@@ -45,8 +45,8 @@ exports.check = function (sr, done) {
         if (!foundRepo) {
```

Comment:
Btw, this will be part of the transition requirements update at https://w3c.github.io/transitions/?docstatus=recommendation